route_info_windows.go

```diff
@@ -22,6 +28,21 @@ func NewRouteInfo() (routeInfo, error) {
 // GetDefaultInterfaceName returns the interface name attached to the default
 // route on the default interface.
 func (ri routeInfo) GetDefaultInterfaceName() (string, error) {
+	if hasPowershell() {
+		ifNameOut, err := exec.Command(cmds["defaultInterface"][0], cmds["defaultInterface"][1:]...).Output()
+		if err != nil {
+			return "", err
+		}
+		ifName := strings.TrimSpace(string(ifNameOut[:]))
+		return ifName, nil
+	} else {
+		return ri.GetDefaultInterfaceNameLegacy()
```

Comment:
A more idiomatic approach would be putting the short block first with an early return:

```go
if !hasPowershell() {
  // No powershell, fallback to legacy method
  return ri.GetDefaultInterfaceNameLegacy()
}

ifNameOut, err := exec.Command(cmds["defaultInterface"][0], cmds["defaultInterface"][1:]...).Output()
if err != nil {
  return "", err
}
ifName := strings.TrimSpace(string(ifNameOut[:]))
return ifName, nil
```